alvarocasadoc/day1/day1.exs

```diff
@@ -0,0 +1,22 @@
+# AdventOfCode2018
+# @alvarocasadoc
+# Day 1
+
+defmodule Day1 do
+
+  # Part 1
+  def sum_frequencies(input) do
+    items = Enum.map(input, fn x -> Integer.parse(x) |> elem(0) end)
+    Enum.sum(items)
+  end
+
+  # Part 2
+  def find_first_cycle(input) do
+    items = Enum.map(input, fn x -> Integer.parse(x) |> elem(0) end)
+    stream = Stream.scan(Stream.cycle(items), &(&1 + &2))
```

Comment:
The `&` thingy is a little bit confusing. I suppose it is some kind of compact lambda and you are just telling to sum elements. Can you substitute that for a function reference like `Stream.scan(..., sum)`?